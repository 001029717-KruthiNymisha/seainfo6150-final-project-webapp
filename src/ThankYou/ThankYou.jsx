import React from 'react';
import styles from "./ThankYou.module.css";
import Thanks from "../images/Thanks.gif";

const ThankYou = () => {
    return (
        <div className={styles.thankYou}>
          <h2>Your order has been successfully placed!</h2>
          <div><img src={Thanks} alt="thanks"/></div>
        </div>
    )
}

export default ThankYou;
