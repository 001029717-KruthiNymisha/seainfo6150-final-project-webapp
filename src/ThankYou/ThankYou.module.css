.thankYou {
  display: grid;
  margin: auto;
  text-align: center;
  padding-left: 22%;
  padding-right: 20%;
  color: black;
  font-size: 20px;
  font-family:cursive;
}
.thankYou img{
width: 90%;
}