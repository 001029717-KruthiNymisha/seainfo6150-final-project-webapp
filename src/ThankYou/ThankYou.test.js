import React from "react";
import { render } from "@testing-library/react";
import ThankYou from "./ThankYou.jsx";

it("renders ThankYou correctly", () => {
  const { container } = render(<ThankYou />);
  expect(container).toMatchSnapshot();
});
