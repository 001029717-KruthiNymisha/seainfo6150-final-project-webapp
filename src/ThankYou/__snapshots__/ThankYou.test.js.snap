// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ThankYou correctly 1`] = `
<div>
  <div
    class="thankYou"
  >
    <h2>
      Your order has been successfully placed!
    </h2>
    <div>
      <img
        alt="thanks"
        src="Thanks.gif"
      />
    </div>
  </div>
</div>
`;
